jackson/src/test/java/com/baeldung/jackson/deserialization/nested/Product.java

```diff
@@ -0,0 +1,80 @@
+package com.baeldung.jackson.deserialization.nested;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(using = ProductDeserializer.class)
+public class Product {
+
+	private String id;
+	private String name;
+	private String brandName;
+	private String ownerName;
+
+	/**

```

Comment:
Let's remove the JavaDoc comments -- they generally aren't needed for getters and setters.